src/main/kotlin/racing/service/GameMachine.kt

```diff
@@ -0,0 +1,32 @@
+package racing.service
+
+import racing.model.Racing
+
+object GameMachine {
+    private const val CAR_SYMBOL = "-"
+    private const val RESULT_SENTENCE = "실행 결과"
+    fun askCounting(question: String): Int {
+        println(question)
+        return readln().orEmpty().toInt()
+    }
+
+    fun printResult(carCount: Int, tryGameCount: Int) {
+        println(RESULT_SENTENCE)
+        play(carCount, tryGameCount)
+    }
+
+    fun play(carCount: Int, tryGameCount: Int) {
+        val racing = Racing(carCount)
+        repeat(tryGameCount) {
+            racing.cars.forEach { car ->
+                val moving = car.run()
+                repeat(moving) {
+                    print(CAR_SYMBOL)
+                }
+                println()
+            }
+            println()
+        }
+        println()
```

Comment:
현재 `Racing` 객체가 아무런 역할도 안하고 있는 것 같아요. 🤔
racing 객체에게 `레이싱 경주를 수행하는` 역할을 부여해보면 어떨까요? 😃

아래와 같이 변경해보시면 좋을 것 같아요. 😉

```suggestion
        val racing = Racing(carCount, tryGameCount)
        racing.play()
```